packages/forms/src/components/Form.story.tsx

```diff
@@ -186,7 +186,7 @@ storiesOf('Forms/Form', module)
         name="date"
         onChange={action('onChange')}
         validator={isDate}
-        defaultValue={fixedDate.toISOString()}
```

Comment:
This was causing the initial value to look broken.

<img width="250" alt="Screen Shot 2019-08-16 at 12 42 26 PM" src="https://user-images.githubusercontent.com/143744/63198104-855cf080-c02e-11e9-9a1d-d5f2dc52c42c.png">

Fixed it by refining the types/parser.

Reply:
👏